src/main/java/com/powsybl/openloadflow/sa/ContingencyParameters.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2024, Coreso SA (https://www.coreso.eu/) and TSCNET Services GmbH (https://www.tscnet.eu/)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.sa;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.contingency.Contingency;
+import com.powsybl.loadflow.LoadFlowParameters;
+
+/**
+ * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
+ */
+public class ContingencyParameters extends AbstractExtension<Contingency> {
```

Comment:
You also need to create a JsonSerializer implementing ExtensionJsonSerializer for this new class.

About the naming, ContingencyLfParameters would maybe be better, to indicate that those are LF parameters? I'm not dead set on it though